languages/python/exercises/concept/comments/.meta/design.md

```diff
@@ -0,0 +1,30 @@
+# Design
+
+## Goal
+
+The goal for this exercise is to teach about comments.
+
+## Things to teach
+
+- How to comment in program.
+
+## Things not to teach
+
+## Concepts
+
+Comment is programmer readable explanation provided by contributer. It makes program more understandable to various stakeholder who involved in that project.
```

Comment:
@aimorris More explanation provided in introduction file.

Reply:
Grammatically incorrect sentence. Should be rewritten.

Reply:
"contributer" should be "contributor" as well.